docs/fava.rst

```diff
@@ -1,10 +1,26 @@
+####
+Fava
+####
 
+:Version: 0.1
+:Date: Oct 27, 2016
 
-### Third Party Rule Plugins ###
+.. contents:: Table of Contents
+    :depth: 6
+
+##############
+Fava Rationale
+##############
+
```

Comment:
@gavin-romig-koch I think it might present better if the sections *Fava Rationale* and *Fava Reference* were subheadings under *Fava*.  This would be consistent with the other major sections of the doc.  See this reference: http://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings